content/logs/attributes_naming_convention.md

```diff
@@ -0,0 +1,139 @@
+| `network.bytes_write` | `string` | Total number of bytes transmitted from the server to the client when the log is emitted. |
+
+Typical integrations relying on these: Apache, Varnish, AWS ELB, Nginx, HAPROXY, etc.
+
+### HTTP Requests
+
+Related to the data commonly used in HTTP requests & accesses. All attributes are prefixed by `http`.
+
+Typical integrations relying on these attributes are: Apache, Rails, AWS CloudFront, web applications servers, etc...
+
+#### Common attributes
+
+|                                                 |                                     |                          |           
+| :---                                            | :---                                 | :----                    |         
+| **Fullname**                                     | **Type**                           | **Description**|
+| `http.url` | `string` | The URL of the HTTP request. |
+| `http.status_code` | `number` | The IP address the client connected to. |
+| `http.method` | `string` | The HTTP verb of the request.|
+| `http.referer` | `string` | The HTTP referer. |
+| `http.request_id` | `string` | The HTTP request id. |
+| `http.user_agent` | `string` | The User-Agent as it is sent (raw format). See bellow for all details about it. |
+
+#### URL details attributes
+
+Details about the parsed parts of the HTTP url. Generally generated thanks to the [URL parser](https://docs.datadoghq.com/logs/processing/#url-parser).
+
+|                                                 |                                     |                          |           
+| :---                                            | :---                                 | :----                    |         
+| **Fullname**                                     | **Type**                           | **Description**|
+| `http.url_details.host` | `string` | The HTTP host part of the url. |
+| `http.url_details.port` | `number` | The HTTP port part of the url. |
+| `http.url_details.path` | `string` | The HTTP path part of the url. |
+| `http.url_details.method` | `string` | The HTTP method part of the url. |
+| `http.url_details.queryString` | `Object` | The HTTP query string parts of the url decomposed as query params key/value attributes. |
+
+#### User-Agent attributes
+
+Details about the meanings of user agents attributes. Generally generated thanks to the [User-Agent parser](https://docs.datadoghq.com/logs/processing/#useragent-parser).
+
+|                                                 |                                     |                          |           
+| :---                                            | :---                                 | :----                    |         
+| **Fullname**                                     | **Type**                           | **Description**|
+| `http.user_agent_details.os.family` | `string` | The OS family reported by the user-agent. |
```

Comment:
Lots of missing attribute, see [here](https://docs.google.com/document/d/1f0A88OaSJ9sFGkRZpwm5uPFITnjYHkwAMhaCSeYmhrI/edit#)  for the missing one

Reply:
I don't think this is worth mentionning the missing attributes